model/flow/protocol_state.go

```diff
@@ -178,17 +210,15 @@ func (e *ProtocolStateEntry) ID() Identifier {
 		return ZeroID
 	}
 	body := struct {
-		CurrentEpochEventIDs            Identifier
 		PreviousEpochEventIDs           Identifier
```

Comment:
I would change this to 
```suggestion
		PreviousEpochEventIDs           EventIDs
```
I don't think the recursive ID call is worth it. If we are sending the data to come client, we only save 32 bytes (one  `Identifier` as opposed to two). But it adds complexity and a second verification step, if the client wants the individual values for `PreviousEpochEventIDs`. For now, I would keep it simple.